diff --git a/makefile b/makefile
index d57ecbad..06e7de7e 100644
--- a/makefile
+++ b/makefile
@@ -58,7 +58,13 @@ ALL_T= $(CORE_T) $(LUA_T) $(LUAC_T)
 ALL_O= $(CORE_O) $(LUA_O) $(LUAC_O) $(AUX_O) $(LIB_O)
 ALL_A= $(CORE_T)
 
-all:	$(ALL_T)
+all:	$(LUA_T)
+
+install: $(LUA_T)
+	mkdir -p $(PREFIX)/bin
+	cp $(LUA_T) $(PREFIX)/bin/
+	mkdir -p $(PREFIX)/include/lua5.1
+	cp *.h $(PREFIX)/include/lua5.1/
 
 o:	$(ALL_O)
 
